deprecated.go

```diff
@@ -0,0 +1,73 @@
+package throttled
```

Comment:
I really like how all of `deprecated` turned out. It's a nice way of organizing all the code that's no longer centrally relevant, and is well-documented in all the right places. I read through and had no questions about it at all.